lte/gateway/c/core/oai/tasks/amf/Registration.cpp

```diff
@@ -691,6 +695,7 @@ int amf_send_registration_accept(amf_context_t* amf_context) {
 
 static int registration_accept_t3550_handler(zloop_t* loop, int timer_id,
                                              void* arg) {
+  OAILOG_FUNC_IN(LOG_NAS_AMF);
```

Comment:
In this function, at some places. LOG_AMF_APP is used. For same function, module name should be same

Reply:
Comment addressed.